Editor/HPUIInteractorPillEditor.cs

```diff
@@ -0,0 +1,65 @@
+using UnityEngine;
+using UnityEditor;
+using ubco.ovilab.HPUI.Interaction;
+
+namespace ubco.ovilab.HPUI.Editor
+{
+    [CanEditMultipleObjects]
+    [CustomEditor(typeof(HPUIInteractorPill), true)]
+    public class HPUIInteractorPillEditor : UnityEditor.Editor
+    {
+        private HPUIInteractorPill t;
+        private bool generating = false;
+        private float maxAngle = 90;
+        private float angleStep = 5;
+        private float axisA = 0.015f;
+        private float axisB = 0.015f;
+        private float axisC = 0.015f;
+
+        protected void OnEnable()
+        {
+            t = target as HPUIInteractorPill;
+        }
+
+        public override void OnInspectorGUI()
+        {
+            base.OnInspectorGUI();
+            if (!generating && GUILayout.Button("Generate new values"))
+            {
+                generating = true;
+            }
+
+            if (generating)
+            {
+                string message = "This will generate the angles.";
+                if (t.angles == null)
+                {
+                    t.angles = new();
+                }
+
+                if (t.angles.Count != 0)
+                {
+                    EditorGUILayout.HelpBox("There are already values configured. Generating would overwrite it!", MessageType.Warning);
+                    message += " There are already values configured. Generating would overwrite it!";
+                }
+                maxAngle = Mathf.Round(EditorGUILayout.Slider("Max Angle", maxAngle, 0, 180));
+                angleStep = Mathf.Round(EditorGUILayout.Slider("Angle Step", angleStep, 0, 180));
+
+                axisA = EditorGUILayout.Slider("Length along local X", axisA, 0, 0.05f);
+                axisB = EditorGUILayout.Slider("Length along local Y", axisB, 0, 0.05f);
+                axisC = EditorGUILayout.Slider("Length along local Z", axisC, 0, 0.05f);
```

Comment:
We need to convey this better maybe something like

along the thumb's length
along the thumb;s lateral thickness
along the thumb;s "axial" thickness? find better wording

Reply:
this is the local vector orientation of the object in Unity, it's clearer that way

Reply:
also brodie I ain't GPT, I'm not about to find better wording

Reply:
you can use gpt tho?

Reply:
i hope?

Reply:
> this is the local vector orientation of the object in Unity, it's clearer that way
I'd like to not think what is x, y and z axis of my thumb when configuring pill
Instead if its clearer like "along the length, lateral radius etc) that is better in editor configuration